liqd_product/templates/base_errors.html

```diff
@@ -8,6 +8,7 @@
     <div class="l-center-6">
         <h1>{% block error_status %}{% endblock %}</h1>
         {% block error_description %}{% endblock %}
+                <a href="/">{% trans 'Back to Beteiligung.in' %}</a>
```

Comment:
Sorry for being more strict than the linter, but could you fix the indentation to be the same everywhere?